library/Fission/Web/User.hs

```diff
@@ -56,6 +58,11 @@ type UpdatePublicKeyRoute
     :> Auth
     :> UpdatePublicKey.API
 
+type UpdateExchangeKeysRoute
+  = "exchange"
```

Comment:
Hmm thinking a bit about naming here. `PUT /user/exchange` is a bit generic (is it a message exchange? Key exchange? Transfer of ownership?). Open to suggestions, but perhaps `PUT /user/did/exchange` or `PUT /user/exchange/keys` or similar?

Reply:
yeah makes sense. I went with `exchange/keys`